1.1/spec/09-Part-06.adoc

```diff
@@ -11,7 +11,7 @@ This clause establishes the *core* requirements class, with URI `/req/core`, whi
 
 ==== Classes
 
-Two main classes are defined: `geo:SpatialObject` and `geo:Feature`. The class `geo:Feature` is equivalent to the UML class `GFI_Feature` defined in [8].
+Three main classes are defined: `geo:SpatialObject`, `geo:SpatialMeasure` and `geo:Feature`. The class `geo:Feature` is equivalent to the UML class `GFI_Feature` defined in [8].
```

Comment:
"The class `geo:Feature` is equivalent to the UML class `Feature` defined in ISO 19109:2015."

Reply:
I've already addressed this one!

Reply:
Fixed with https://github.com/opengeospatial/ogc-geosparql/pull/72/commits/17073e57566de6e6773919f1af6fed9cadde0cde